modules/javafx.web/src/main/native/Source/ThirdParty/icu/CMakeLists.txt

```diff
@@ -588,7 +590,7 @@ target_link_libraries(data_as_asm icuuc icui18n)
 file(GLOB ICU_DATA_ZIP_FILE "${CMAKE_BINARY_DIR}/icu/data/icu4c-*.zip")
 
 # Get just file name without extension
-set(ICU_DATA_FILE_NAME "icudt73l")
+set(ICU_DATA_FILE_NAME "icudt74l")
```

Comment:
Shouldn't this be 742?

Reply:
Never mind. I see now that's an `l` not a `1` after the `74. Sorry for the noise.

Reply:
icu4c-74_2-data-bin-l contains the dat file named as icudt74l.dat. Hence, this is correct.